wgengine/filter/filter_test.go

```diff
@@ -39,11 +40,13 @@ func m(srcs []netip.Prefix, dsts []NetPortRange, protos ...ipproto.Proto) Match
 	if protos == nil {
 		protos = defaultProtos
 	}
-	return Match{
+	m := Match{
 		IPProto: protos,
 		Srcs:    srcs,
 		Dsts:    dsts,
 	}
```

Comment:
you could do `SrcsContains: tsaddr.NewContainsIPFunc(views.SliceOf(srcs))`

Reply:
Done.